app/views/govuk_publishing_components/components/_column_list.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  title ||= nil
+  title_hidden ||= false
+  items ||= []
+%>
+<% if items.any? %>
+  <% if title.present? %>
+    <h2 class="<%= title_hidden ? "govuk-visually-hidden" : "govuk-heading-m" %>"><%= title %></h2>
```

Comment:
I was neutral on this when we discussed it previously but I'm now thinking this belongs outside the component. A page with hidden headings is probably going to have more than one and we don't make it a part of any other component. Additionally you might want a heading that isn't hidden, so you'd have to extend this component or deliberately not feed your data into it so you could present a heading separately. It just seems like extra overhead that doesn't contribute to what the component should be focusing on - making a list lay out horizontally.

Reply:
Good shout, the title makes the component too specific. I'll scoop it out.